manager/src/main/resources/application.properties

```diff
@@ -1,6 +1,13 @@
 # Swagger and openapi
 quarkus.swagger-ui.always-include=true
 quarkus.smallrye-openapi.store-schema-directory=target/generated/
+quarkus.smallrye-openapi.info-title=Red Hat Openshift Smart Events Fleet Manager
+quarkus.smallrye-openapi.info-version=0.0.1
+quarkus.smallrye-openapi.info-description=The api exposed by the fleet manager of the RHOSE service.
+quarkus.smallrye-openapi.info-license-name=Apache 2.0
+quarkus.smallrye-openapi.info-license-url=https://www.apache.org/licenses/LICENSE-2.0.html
+quarkus.smallrye-openapi.info-contact-email=openbridge-dev@redhat.com
```

Comment:
Do we already have `rhose-dev` mailing list?

Reply:
I think we only have `rhose`. @danielezonca or @robpblake can you create it?